### ISSUE TITLE
Alternative solution for question 4

### ISSUE THREAD

**Issue by daviddoji** [rougier/numpy-100#157]
One could use simply `nbytes` to get an alternative solution to question 4.
```
>>> Z = np.zeros((10,10))
>>> print(f"{Z.nbytes} bytes")
800 bytes
```

**Comment by rougier** [comment]
You're right. Can you make a PR (maybe just adding s second argument in the print with the nbytes). I want to keep itemsize and size such that people can see these attributes exist.
